model/task/task.go

```diff
@@ -153,8 +153,10 @@ type Task struct {
 	GeneratedTasks bool `bson:"generated_tasks,omitempty" json:"generated_tasks,omitempty"`
 	// GeneratedBy, if present, is the ID of the task that generated this task.
 	GeneratedBy string `bson:"generated_by,omitempty" json:"generated_by,omitempty"`
-	// GeneratedJSON is the the configuration information to create new tasks from.
+	// GeneratedJSONKey is no longer used but must be kept for old tasks.
 	GeneratedJSON []json.RawMessage `bson:"generate_json,omitempty" json:"generate_json,omitempty"`
+	// GeneratedJSONAsString is the configuration information to create new tasks from.
+	GeneratedJSONAsString []string `bson:"generated_json,omitempty" json:"generated_json,omitempty"`
```

Comment:
Why not keep this all in one field and write a custom marshaler that does the right thing with either input.

Reply:
I believe that strategy isn't backwards compatible (I think that's why we implemented ParamsYAML for ParserProject, and also why I think these bson tags need to be different)

Reply:
I think that bson is different than yaml, because it has more type data that you can access when writing the marshaler?

Reply:
But if the marshaler is written as part of this ticket then in case of a revert will we know how to handle a string? What if the string somehow isn't correct and now the only populated field is corrupted?